### PR TITLE
Shaokai/fix

### PR DESCRIPTION
**Pull request by yeshaokai**
Added suggestions from Niels

### CODE REVIEW

File: notebooks/EPM_demo.ipynb [hunk 1]
```diff
@@ -51,6 +51,8 @@
     "amadeus_root = Path(amadeusgpt.__file__).parent.parent\n",
     "config = Config(amadeus_root / \"amadeusgpt/configs/EPM_template.yaml\")\n",
     "config['video_info']['scene_frame_number'] = scene_frame_number\n",
+    "# if you install from pypl or you used the conda file, you have to adjust the path of data folder to\n",
```
Comment (MMathisLab) [inline notebooks/EPM_demo.ipynb:54]: Typo: "pypi"

File: notebooks/Horse_demo.ipynb [hunk 2]
```diff
@@ -41,6 +41,8 @@
     "config = Config(amadeus_root / \"amadeusgpt/configs/Horse_template.yaml\")\n",
     "\n",
     "config['video_info']['scene_frame_number'] = scene_frame_number\n",
+    "# if you install from pypl or you used the conda file, you have to adjust the path of data folder to\n",
```
Comment (MMathisLab) [inline notebooks/Horse_demo.ipynb:44]: Typo pypi

File: notebooks/MABe_demo.ipynb [hunk 3]
```diff
@@ -38,6 +38,8 @@
    "source": [
     "amadeus_root = Path(amadeusgpt.__file__).parent.parent\n",
     "config = Config(amadeus_root / \"amadeusgpt/configs/MABe_template.yaml\")\n",
+    "# if you install from pypl or you used the conda file, you have to adjust the path of data folder to\n",
```
Comment (MMathisLab) [inline notebooks/MABe_demo.ipynb:41]: Typo pypi

File: notebooks/MausHaus_demo.ipynb [hunk 4]
```diff
@@ -43,7 +43,8 @@
     "config = Config(amadeus_root / \"amadeusgpt/configs/MausHaus_template.yaml\")\n",
     "\n",
     "config['video_info']['scene_frame_number'] = scene_frame_number\n",
-    "\n",
+    "# if you install from pypl or you used the conda file, you have to adjust the path of data folder to\n",
```
Comment (MMathisLab) [inline notebooks/MausHaus_demo.ipynb:46]: Typo pypi

**Review by MMathisLab — CHANGES_REQUESTED** [review]

**Comment by yeshaokai** [comment]
@MMathisLab Fixed typos

**Review by MMathisLab — APPROVED** [review]

**Pull request merged**